07-routing-gossip.md

```diff
@@ -170,12 +172,12 @@ nodes for which a channel is not already known are ignored.
 1. type: 257 (`node_announcement`)
 2. data:
    * [`64`:`signature`]
+   * [`2`:`flen`]
+   * [`flen`:`features`]
```

Comment:
Similar comment here about moving the feature bits to the _very top_ of the message.